### PR TITLE
be able to specify the console version we want

### PR DESCRIPTION
**Pull request by jmazzitelli**
related to PR #61

### CODE REVIEW

**Comment by jmazzitelli** [comment]
this was a suggestion from @josejulio

**Comment by jmazzitelli** [comment]
this also adds the feature suggested by @pilhuhn - if the console version is "local" we don't download it from the npm repo but instead we just copy the /build/* files from your locally build swsui project. 

`CONSOLE_VERSION=local make docker`

By default it assumes swsui is located as a peer directory to the directory that is the root of your swscore GOPATH. So it assumes a file structure:

/some-root-dir-like-swift-sunshine/swsui/build/...the build files here...
/some-root-dir-like-swift-sunshine/swscore/src/github.com/swift-sunshine/swscore/...git repo of source here...

If your SWSUI is somewhere that doesn't match that relationship with swscore, then you set CONSOLE_LOCAL_DIR to point to your local git repo for swsui like this:

`CONSOLE_VERSION=local CONSOLE_LOCAL_DIR=/my/git/repo/for/swsui make docker`

The only requirement for the value of that local_dir env var is that it has a subdirectory "build" which is where the swsui build files are (i.e. `${CONSOLE_LOCAL_DIR}/build`)

**Comment by jmazzitelli** [comment]
To pull in the latest UI console from npm repo, nothing extra needs to be specified - its just `make docker`

To pull in a specific version it is: `CONSOLE_VERSION=0.2.3 make docker`

File: Makefile [hunk 1]
```diff
@@ -76,32 +84,41 @@ dep-update:
...
+.get-console:
 	@mkdir -p _output/docker
-	@cp -r deploy/docker/* _output/docker
-	@cp ${GOPATH}/bin/sws _output/docker
+ifeq ("${CONSOLE_VERSION}", "local")
+	echo "Copying local console files from ${CONSOLE_LOCAL_DIR}"
+	rm -rf _output/docker/console && mkdir _output/docker/console
+	cp -r ${CONSOLE_LOCAL_DIR}/build/* _output/docker/console
+else
 	@if [ ! -d "_output/docker/console" ]; then \
```
Comment (josejulio) [inline Makefile]: I think this is to cache the previous download, but we will need something to know if `_output/docker/console` holds the value of specified ${CONSOLE_VERSION} or some old value (local code, previous version, etc).

```bash
# Make docker with local
CONSOLE_VERSION=local make docker
# Try to make docker with version=0.2.3  In reality we are using our old _output/docker/console dir
CONSOLE_VERSION=0.2.3 make docker
```

Comment (jmazzitelli) [reply Makefile]: yeah, that is a shortcoming. You have to make clean if you want to switch versions. I didn't bother - I assume the user will know when he is switching between versions and will know to clean. For now, I thought this was good enough. I didn't want to have to code up everything to figure out automatically if the versions matched

Comment (josejulio) [reply Makefile]: I ok with that, but warning on the comments above `CONSOLE_VERSION ?= latest` would be great.

Comment (jmazzitelli) [reply Makefile]: comment added - see the commit #7565c41

**Comment by josejulio** [comment]
Can’t merge, I don’t have permissions :)

**Pull request merged**